src/resolve/resolved-dns-packet.c

```diff
@@ -2269,6 +2269,9 @@ int dns_packet_is_reply_for(DnsPacket *p, const DnsResourceKey *key) {
         if (r < 0)
                 return r;
 
+        if (!p->question)
```

Comment:
yeah, our codebase isn't really clean in this regard... But for new code we try to follow the same style.

That said we aren't overlay strict on this really anyway. The 8ch indenting otoh matters more